### PR TITLE
[wip] Bump dep versions.

### PR DESCRIPTION
**Pull request by brndnmtthws**

### CODE REVIEW

File: src/main/scala/org/apache/mesos/chronos/scheduler/jobs/stats/JobStats.scala [hunk 1]
```diff
@@ -190,7 +193,9 @@ class JobStats @Inject()(clusterBuilder: Option[Cluster.Builder], config: Cassan
    * @return updated TaskStat object
    */
   private def updateTaskStat(taskStat: TaskStat, row: Row): TaskStat = {
-    val taskTimestamp = row.getDate(TIMESTAMP)
+    val localDate = row.getDate(TIMESTAMP)
+    calendar.set(localDate.getYear, localDate.getMonth, localDate.getDay)
+    val taskTimestamp = calendar.getTime
```
Comment (ddossot) [inline src/main/scala/org/apache/mesos/chronos/scheduler/jobs/stats/JobStats.scala]: This change doesn't seem related to bumping dep versions. Should probably be in another pull request.

Comment (brndnmtthws) [inline src/main/scala/org/apache/mesos/chronos/scheduler/jobs/stats/JobStats.scala]: I did this to address compiler warnings.

Comment (ddossot) [inline src/main/scala/org/apache/mesos/chronos/scheduler/jobs/stats/JobStats.scala]: Alright :relieved: Sorry, I'm easily panicked by changes to time stuff :sweat_smile:

**Comment by brndnmtthws** [comment]
I still need to test this properly before it gets merged.

**Comment by timcharper** [comment]
I'd really like to see the Cassandra driver get upgraded; driver `3.0.0` is compatible with Cassandra `2.x.x`, but driver `2.x.x` is not compatible with `3.x.x`. http://datastax.github.io/java-driver/manual/native_protocol/
